azure-mgmt-keyvault/src/main/java/com/microsoft/azure/management/keyvault/CheckNameAvailabilityResult.java

```diff
@@ -0,0 +1,16 @@
+package com.microsoft.azure.management.keyvault;
+
+import com.microsoft.azure.management.apigeneration.Fluent;
+import com.microsoft.azure.management.keyvault.implementation.CheckNameAvailabilityResultInner;
+import com.microsoft.azure.management.resources.fluentcore.model.HasInner;
+
+/**
+ * Wrapper for CheckNameAvailabilityResultInner
+ * @author tifchen
+ *
+ */
+@Fluent(ContainerName = "/Microsoft.Azure.Management.Fluent.KeyVault")
+public interface CheckNameAvailabilityResult extends
+    HasInner<CheckNameAvailabilityResultInner>{
+

```

Comment:
I would expect to see here some methods such as isAvailable(), unavailabilityReason() and unavailabilityMessage()...

Reply:
Added in latest commit